scripts/ci/run_perftest.sh

```diff
@@ -117,36 +156,78 @@ for scale in 50 100 500; do
     # run ingest tests
     cat ${DATASET_DIR}/$data_fname | $GOPATH/bin/bulk_load_influx $load_opts | jq ". += {branch: \"${INFLUXDB_VERSION}\", commit: \"${TEST_COMMIT}\", time: \"$datestring\", i_type: \"${DATA_I_TYPE}\"}" > $working_dir/test-ingest-$scale_string-batchsize-$batch-workers-$workers.json
 
-    bucket_id=$(curl -H "Authorization: Token $TEST_TOKEN" "http://${NGINX_HOST}:8086/api/v2/buckets?org=$TEST_ORG" | jq -r ".buckets[] | select(.name | contains(\"$db_name\")).id")
-    sleep 1
-    # stop daemon and force compaction
-    systemctl stop influxdb
-    set +e
-    shards=$(find /var/lib/influxdb/engine/data/$bucket_id/autogen/ -maxdepth 1 -mindepth 1)
-    set -e
-    for shard in $shards; do
-      /home/ubuntu/influx_tools compact-shard -force -verbose -path $shard
-    done
-
-    # restart daemon
-    systemctl unmask influxdb.service
-    systemctl start influxdb
+    force_compaction
 
     # run influxql and flux query tests
     for query_file in $query_files; do
       format=$(echo $query_file | cut -d '-' -f7)
+      format=${format%.txt}
```

Comment:
Some of the formats would include the file extension with the existing logic; this will drop the file extension of `.txt` if it is present